pkg/nsx/services/common/types.go

```diff
@@ -28,38 +28,53 @@ const (
 	TagScopeRuleID                  string = "nsx-op/rule_id"
 	TagScopeGroupType               string = "nsx-op/group_type"
 	TagScopeSelectorHash            string = "nsx-op/selector_hash"
-	TagScopeNSXServiceAccountCRName string = "nsx-op/nsx_service_account_name"
-	TagScopeNSXServiceAccountCRUID  string = "nsx-op/nsx_service_account_uid"
+	TagScopeIPPoolCRName            string = "nsx-op/ippool_cr_name"
+	TagScopeIPPoolCRUID             string = "nsx-op/ippool_cr_uid"
+	TagScopeIPPoolCRType            string = "nsx-op/ippool_cr_type"
+	TagScopeIPPoolCreatedFor        string = "nsx-op/ippool_created_for"
+	TagScopeIPSubnetName            string = "nsx-op/ipsubnet_cr_name"
+	TagScopeIPSubnetCreatedFor      string = "nsx-op/ipsubnet_created_for"
```

Comment:
can we use nsx-op/created_for as tag scope ?

Reply:
ok